src/ert/dark_storage/common.py

```diff
@@ -132,6 +132,17 @@ def _get_obs_data(key, obs) -> dict:
 
 
 def _prepare_x_axis(x_axis: List[Union[int, float, str, pd.Timestamp]]) -> List[str]:
+    """Converts the elements of x_axis of an observation to a string suitable
+    for json.
+
+    >>> _prepare_x_axis([1, 2, 3, 4])
+    ['1', '2', '3', '4']
+
+    The x_axis can possibly be a timestamp in which case isoformat is used
```

Comment:
Can delete this if you accept previous change.

Reply:
Done